### ISSUE TITLE
Key Mapping is incorrect

### ISSUE THREAD

**Issue by vihanb** [jQueryOS/jQueryOS#2]
When a key is pressed, the incorrect character is displayed.

For example pressing `c` prints `5`

![](http://i.stack.imgur.com/xMS3L.gif)

Reproduced by @RikerW and me.

**Comment by phase** [comment]
You'll need to manually setup a character map, as all the codes are different. [Here's](https://github.com/phase/OS/blob/master/kernel/kernel.c#L22) my implementation.

**Comment by RikerW** [comment]
@phase is that your own OS project, unrelated to this one?

**Comment by phase** [comment]
@RikerW Yes, it's one I made a while ago, just following the Bare Bones example and other stuff from osdev. My main one is private source, and it has a lot more work on it.

**Comment by somebody1234** [comment]
Should be fixed in dev branch.
